api/src/test/java/com/epam/pipeline/security/saml/SAMLUserDetailsServiceImplTest.java

```diff
@@ -220,6 +220,32 @@ public void shouldThrowAuthorizationExceptionForBlockedUser() {
         userDetailsService.loadUserBySAML(credential);
     }
 
+    @Test
+    public void testBlockedValueCaseNonSensitiveness() {
```

Comment:
Also, could you please add a positive tests? (if attribute does not specified/ if attribute is equal to false e.g.)

Reply:
Done